src/hooks/use-archive-site.ts

```diff
@@ -109,6 +111,11 @@ export function useArchiveSite() {
 					formData.push( [ 'wordpress_version', wordpressVersion ] );
 				}
 
+				const connectedProductionSite = connectedSites.find( ( site ) => ! site.isStaging );
+				if ( connectedProductionSite ) {
+					formData.push( [ 'connected_site_id', String( connectedProductionSite.id ) ] );
```

Comment:
Why are we casting the Int to String?
I've seen that in 170864-ghe-Automattic/wpcom/files#diff-2fce9fc365683142c30cb822e249f39d53d53ca73e431da75987e2622349d43dR240 we are casting it back to int again. I wonder if it makes sense using just the default type Int in the first place.

```suggestion
					formData.push( [ 'connected_site_id', connectedProductionSite.id ] );
```

Reply:
Good catch! I have fix the issue in https://github.com/Automattic/studio/pull/838/commits/03e0ecbbb786ee962222becca8dcc3d35fd964d4.